### PR TITLE
docs: update wording for gateway.md

### PR DESCRIPTION
**Pull request by CaitinChen**
Via https://github.com/juicedata/juicefs/pull/5353

### CODE REVIEW

File: docs/en/guide/gateway.md [hunk 1]
```diff
@@ -577,12 +577,12 @@ USAGE:
...
 
 COMMANDS:
-  add      add users to a new or existing group
-  remove   remove group or members from a group
-  info     display group info
-  list     display list of groups
-  enable   enable a group
-  disable  disable a group
+  add      Add users to a new or existing group
+  remove   Remove group or members from a group
```
Comment (CaitinChen) [inline docs/en/guide/gateway.md:581]:
```suggestion
  remove   Remove a group or its members
```

File: docs/en/guide/gateway.md [hunk 2]
```diff
@@ -577,12 +577,12 @@ USAGE:
...
 
 COMMANDS:
-  add      add users to a new or existing group
-  remove   remove group or members from a group
-  info     display group info
-  list     display list of groups
-  enable   enable a group
-  disable  disable a group
+  add      Add users to a new or existing group
+  remove   Remove a group or its members
```
Comment (CaitinChen) [inline docs/en/guide/gateway.md:581]: @timfeirg PTAL

File: docs/en/guide/gateway.md [hunk 3]
```diff
@@ -607,19 +607,19 @@ FILE:
...
-  2. Set bucket to "public" on Amazon S3 cloud storage.
+  2. Set a bucket to "public" on Amazon S3 cloud storage.
      $ mc policy set public s3/shared
 
-  3. Set bucket to "upload" on Amazon S3 cloud storage.
+  3. Set a bucket to "upload" on Amazon S3 cloud storage.
      $ mc policy set upload s3/incoming
 
-  4. Set policy to "public" for bucket with prefix on Amazon S3 cloud storage.
+  4. Apply a "public" policy to a specific bucket prefix on Amazon S3 cloud storage.
```
Comment (CaitinChen) [inline docs/en/guide/gateway.md:619]: @timfeirg PTAL. I'm not sure what this sentence means.

File: docs/en/guide/gateway.md [hunk 4]
```diff
@@ -607,19 +607,19 @@ FILE:
...
      $ mc policy set public s3/public-commons/images
 
-  5. Set a custom prefix based bucket policy on Amazon S3 cloud storage using a JSON file.
+  5. Assign a custom policy to a bucket on Amazon S3 cloud storage using a JSON file.
```
Comment (CaitinChen) [inline docs/en/guide/gateway.md:622]: @timfeirg PTAL. I'm not sure what this sentence means.

File: docs/en/guide/gateway.md (hunk 2 above)
Comment (timfeirg) [reply docs/en/guide/gateway.md:581]: this content is extracted directly from minio client output, do not modify @CaitinChen

File: docs/en/guide/gateway.md [hunk 5]
```diff
@@ -495,13 +495,13 @@ USAGE:
...
-  set     set IAM policy on a user or group
-  unset   unset an IAM policy for a user or group
-  update  Attach new IAM policy to a user or group
+  add     Add a new policy
+  remove  Remove a policy
+  list    List all policies
+  info    Show info on a policy
+  set     Set IAM policy on a user or group
+  unset   Unset an IAM policy for a user or group
+  update  Attach a new IAM policy to a user or group
```
Comment (timfeirg) [inline docs/en/guide/gateway.md:504]: this content is extracted directly from minio client output, do not modify @CaitinChen

File: docs/en/guide/gateway.md [hunk 6]
```diff
@@ -705,10 +699,10 @@ To reduce dependencies, JuiceFS S3 Gateway has cut support for certain event des
...
 ```Shell
 $ mc admin config get myjfs | grep notify
-notify_webhook        publish bucket notifications to webhook endpoints
-notify_mysql          publish bucket notifications to MySQL databases
-notify_postgres       publish bucket notifications to Postgres databases
-notify_redis          publish bucket notifications to Redis datastores
+notify_webhook        Publish bucket notifications to webhook endpoints
+notify_mysql          Publish bucket notifications to MySQL databases
+notify_postgres       Publish bucket notifications to Postgres databases
+notify_redis          Publish bucket notifications to Redis datastores
```
Comment (timfeirg) [inline docs/en/guide/gateway.md:705]: same

**Pull request merged**